firebase.incl.json

```diff
@@ -7,7 +7,7 @@
         "headers": [
           {
             "key": "Cache-Control",
-            "value": "public,max-age=31536000,immutable"
+            "value": "max-age=31536000"
```

Comment:
I don't think `public` will matter, because I think Bleigh said that we can not set cache control for static content on the CDN. As far as `immutable` is concerned, the content isn't immutable, right? Technically it does change over time.